app/views/application/sdip/considerMeForSdip.scala.html

```diff
@@ -0,0 +1,37 @@
+@(form: Form[_root_.forms.SdipForm.Data])(implicit request: Request[_], flash: Flash, user: Option[CachedData], feedbackUrl: String)
+
+@import views.html.widgets.checkbox
+
+@main_template(title = "Consider me for SDIP", pageForms = Seq(form)) {
+    <div class="hgroup">
+        <h1 class="heading-xlarge">Consider me for SDIP</h1>
+        <p class="subtitle">For existing Civil Service Fast Stream applicants</p>
+    </div>
+    @helper.form(action = routes.SdipController.submit(), 'novalidate -> "novalidate") {
```

Comment:
we need to fix this later. I'll wait for more method (after the enhancment story) and then we will know if we can do this